src/main/java/upb/dice/twitter/SearchTweets.java

```diff
@@ -0,0 +1,47 @@
+package upb.dice.twitter;
+
+import twitter4j.GeoLocation;
+import twitter4j.GeoQuery;
+import twitter4j.Query;
+
+import java.io.IOException;
+
+/**
+ * A query is built based on the keyword or location provided and tweets related to this query are extracted
+ */
+public class SearchTweets {
```

Comment:
This is not really a class name. It looks more like a name for a function or a method.